<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QTableWidget" name="student_table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>98</y>
     <width>592</width>
     <height>361</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-radius: 10px;
background-color: #CAC4CE;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerItem</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::NoPen</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>44</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>118</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NAME</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PROGRAM</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>YEAR LEVEL</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>STATUS</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="search_tab">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>295</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
	border: none;
	border-radius: 5px;
	background-color: #9067C6;
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>SEARCH</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>30</y>
     <width>350</width>
     <height>433</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #F7ECE1;
border: none;
border-radius: 10px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="student_title">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NEW STUDENT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="create_button">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>350</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 5px;
	background-color: #9067C6;
}

QPushButton::hover {
	background-color: #431480;
	color: white;
}</string>
    </property>
    <property name="text">
     <string>CREATE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 5px;
	background-color: #F4D35E;
}

QPushButton::hover {
	background-color: #C1A74C;
	color: white;
}</string>
    </property>
    <property name="text">
     <string>CANCEL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>53</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="last_name">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>295</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
	border: none;
	border-radius: 5px;
	background-color: #9067C6;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Last Name*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="first_name">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>295</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
	border: none;
	border-radius: 5px;
	background-color: #9067C6;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>First Name*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="middle_name">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>295</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
	border: none;
	border-radius: 5px;
	background-color: #9067C6;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Middle Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gender">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>295</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
	border: none;
	border-radius: 5px;
	background-color: #9067C6;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Gender*</string>
    </property>
   </widget>
   <widget class="QComboBox" name="year_level">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>295</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	border-radius: 5px;
	background-color: #9067C6;
	border: none;
}
</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string notr="true">Course*</string>
    </property>
    <item>
     <property name="text">
      <string>Year Level*</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1st Year</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2nd Year</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3rd Year</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4th Year</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="course">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>295</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	border-radius: 5px;
	background-color: #9067C6;
	border: none;
}
</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="placeholderText">
     <string notr="true">Course*</string>
    </property>
    <item>
     <property name="text">
      <string>Course*</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>No Course</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 5px;
	background-color: #9067C6;
}

QPushButton::hover {
	background-color: #431480;
	color: white;
}</string>
    </property>
    <property name="text">
     <string>SAVE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edit_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 5px;
	background-color: #9067C6;
}

QPushButton::hover {
	background-color: #431480;
	color: white;
}</string>
    </property>
    <property name="text">
     <string>EDIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 5px;
	color: white;
	background-color: #FF0000;
}

QPushButton::hover {
	background-color: #C90000;
	color: white;
}</string>
    </property>
    <property name="text">
     <string>DELETE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_year">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
	border: none;
	border-radius: 5px;
	background-color: #9067C6;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Year</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_number">
    <property name="geometry">
     <rect>
      <x>187</x>
      <y>50</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
	border: none;
	border-radius: 5px;
	background-color: #9067C6;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>IDNo</string>
    </property>
   </widget>
   <zorder>save_button</zorder>
   <zorder>delete_button</zorder>
   <zorder>student_title</zorder>
   <zorder>cancel_button</zorder>
   <zorder>label</zorder>
   <zorder>last_name</zorder>
   <zorder>first_name</zorder>
   <zorder>middle_name</zorder>
   <zorder>gender</zorder>
   <zorder>year_level</zorder>
   <zorder>course</zorder>
   <zorder>id_year</zorder>
   <zorder>id_number</zorder>
   <zorder>edit_button</zorder>
   <zorder>create_button</zorder>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>70</y>
     <width>60</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>70</y>
     <width>60</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>NAME</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Year Level</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Program</string>
   </property>
  </widget>
  <widget class="QPushButton" name="new_button">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: none;
	border-radius: 5px;
	background-color: #9067C6;
}

QPushButton::hover {
	background-color: #431480;
	color: white;
}</string>
   </property>
   <property name="text">
    <string>NEW</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>523</x>
     <y>70</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="no_students">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>No students. Try Adding One!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
